src/sentry/static/sentry/app/views/organizationIntegrations/integrationStatus.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import CircleIndicator from 'app/components/circleIndicator';
+import theme from 'app/utils/theme';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {withTheme} from 'emotion-theming';
+
+import {colors} from './constants';
+
+type Props = {
+  status?: string;
+  enabled?: boolean;
+  theme?: any;
```

Comment:
@NisanthanNanthakumar Is `theme` really a prop that's being passed down here? I think you only need it in `StatusProps` since it's being wrapped with `withTheme`.